java/arcs/core/entity/Reference.kt

```diff
@@ -32,6 +32,11 @@ class Reference<T : Entity>(
     val expirationTimestamp
         get() = storageReference.expirationTimestamp
 
+    val isHardReference
+        get() = storageReference.isHardReference
+
+    fun setHardReference() { storageReference.isHardReference = true }
```

Comment:
Nit: don't one-line method bodies like this.. The only time they should be one-lined is if they're able to be expressed using function-expression notation.
```suggestion
    fun setHardReference() { 
      storageReference.isHardReference = true 
    }
```